tests/integration/telemetry/stackdriver/vm/testdata/server_edge.prototext.tmpl

```diff
@@ -0,0 +1 @@
+source:<owner_uid:"kubernetes://apis/apps/v1/namespaces/{{ .EchoNamespace }}/deployments/client-v1" workload_name:"client-v1" workload_namespace:"{{ .EchoNamespace }}" canonical_service:"client" canonical_revision:"v1" > destination:<location:"us-west1-c" owner_uid:"//compute.googleapis.com/some-creator" workload_name:"vm-server-v1" workload_namespace:"{{ .EchoNamespace }}" canonical_service:"vm-server" canonical_revision:"v1" > protocol:PROTOCOL_HTTP destination_service_name:"server" destination_service_namespace:"{{ .EchoNamespace }}" 
```

Comment:
Could you format this into multi-line text? It's hard to read a long line.

Reply:
Done.